test_pull_requests

```diff
@@ -412,8 +412,8 @@ module Hub
 
       repo_to_pull_request.each do |repo, pull_request|
         if repo != base_repo
-          process_or_create_comment(pull_request['number'], repo, settings) do |comment_id, comment, comment_updated_at|
-            update_comment(comment_id, repo, running_comment)
+          process_or_create_comment(pull_request['number'], repo, settings) do |bot_comment_id, comment, comment_updated_at|
```

Comment:
Ah sorry, I misread and missed the do.  How about using c_id.  Since both are bot_comment_ids.